### PR TITLE
Filter credentials by issuer DID, circut urls

### PR DESCRIPTION
**Pull request by lukachi**
- Explicitly filter credentials by `issuerDid` when generating ZKPs;
- Use new circuit ipfs urls to improve performance;

### CODE REVIEW

File: packages/snap/src/index.ts [hunk 1]
```diff
@@ -156,8 +158,12 @@ export const onRpcRequest: OnRpcRequestHandler = async ({
...
       }
 
-      const credentials = (await findCredentialsByQuery(params.query)).filter(
-        (cred) => cred.credentialSubject.id === identityStorage.did,
+      const credentials = (
+        await findCredentialsByQuery(createProofRequest.query)
+      ).filter(
+        (cred) =>
+          cred.credentialSubject.id === identityStorage.did &&
+          cred.issuer === issuerDID,
```
Comment (lukachi) [inline packages/snap/src/index.ts]: should we lowercase it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.7.0] - 2023-10-20
+### Changed
```
Comment (ihordiachenko) [inline CHANGELOG.md:11]:
```suggestion
### Fixed
```

File: packages/snap/src/types/proof-types.ts [hunk 3]
```diff
@@ -22,6 +22,10 @@ export type CreateProofRequest = {
   query: ProofQuery;
 };
 
+export type CreateProofRequestParams = {
+  issuerDID: string;
```
Comment (ihordiachenko) [inline packages/snap/src/types/proof-types.ts]:
```suggestion
  issuerDid: string;
```

For consistency sake

File: packages/connector/src/types.ts [hunk 4]
```diff
@@ -94,6 +94,7 @@ export type ProofQuery = {
 export type CreateProofRequestParams = {
   id?: number;
   accountAddress?: string; // Metamask user address for onchain proofs
+  issuerDID: string;
```
Comment (ihordiachenko) [inline packages/connector/src/types.ts]: Update the docs too

File: packages/snap/src/index.ts (hunk 1 above)
Comment (ihordiachenko) [reply packages/snap/src/index.ts]: No, base58 is case sensetive

File: packages/snap/README.md [hunk 5]
```diff
@@ -72,6 +72,7 @@ await window.ethereum.request({
       method: 'create_proof',
       params: {
         circuitId: 'credentialAtomicQuerySigV2OnChain',
+        issuerDid: 'did:iden3:[...]',
```
Comment (ihordiachenko) [inline packages/snap/README.md:75]: Also add a description for this param below

**Review by ihordiachenko — APPROVED** [review]

**Pull request merged**